packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -240,6 +245,9 @@ export default function Sidebar({
     return <></>;
   }
 
+  const shouldHideMyFeedAlert =
+    alerts?.filter || (!alerts?.filter && alerts?.myFeed === null);
```

Comment:
I suggest this. Otherwise, if alerts is undefined this condition will be true.
What we actually want to check is that it is not set to any actual value.

```suggestion
    alerts?.filter || (!alerts?.filter && !!alerts?.myFeed);
```